api/routes/web.py

```diff
@@ -0,0 +1,95 @@
+"""
+Web routes
+"""
+from typing import Optional, List
+
+from fastapi import APIRouter, Request
+from pydantic import BaseModel
+
+from models.models.sample import sample_id_format
+from db.python.layers.web import WebLayer, NestedParticipant
+
+from api.utils.db import (
+    get_project_readonly_connection,
+    Connection,
+)
+
+
+class PagingLinks(BaseModel):
+    """Model for PAGING"""
+
+    self: str
+    next: Optional[str]
+    token: Optional[str]
+
+
+class ProjectSummaryResponse(BaseModel):
+    """Response for the project summary"""
+
+    participants: List[NestedParticipant]
+    total_samples: int
+    sample_keys: List[str]
+    sequence_keys: List[str]
+
+    links: Optional[PagingLinks]
+
+    class Config:
```

Comment:
Where is the Config class used?

Reply:
It's used by pydantic to configure the serialization.